proto/pdpb2.proto

```diff
@@ -0,0 +1,314 @@
+syntax = "proto3";
+package pdpb2;
+
+import "metapb.proto";
+import "eraftpb.proto";
+
+import "gogoproto/gogo.proto";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+
+service PD {
+    // GetPDMembers get the member list of this cluster. It does not require
+    // the cluster_id in request matchs the id of this cluster.
+    rpc GetPDMembers(GetPDMembersRequest) returns (GetPDMembersResponse) {}
+
+    rpc Tso(TsoRequest) returns (TsoResponse) {}
+
+    rpc Bootstrap(BootstrapRequest) returns (BootstrapResponse) {}
+
+    rpc IsBootstrapped(IsBootstrappedRequest) returns (IsBootstrappedResponse) {}
+
+    rpc AllocID(AllocIDRequest) returns (AllocIDResponse) {}
+
+    rpc GetStore(GetStoreRequest) returns (GetStoreResponse) {}
+
+    rpc PutStore(PutStoreRequest) returns (PutStoreResponse) {}
+
+    rpc StoreHeartbeat(StoreHeartbeatRequest) returns (StoreHeartbeatResponse) {}
+
+    rpc RegionHeartbeat(RegionHeartbeatRequest) returns (RegionHeartbeatResponse) {}
+
+    rpc GetRegion(GetRegionRequest) returns (GetRegionResponse) {}
+
+    rpc GetRegionByID(GetRegionByIDRequest) returns (GetRegionResponse) {}
+
+    rpc AskSplit(AskSplitRequest) returns (AskSplitResponse) {}
+
+    rpc ReportSplit(ReportSplitRequest) returns (ReportSplitResponse) {}
+
+    rpc GetClusterConfig(GetClusterConfigRequest) returns (GetClusterConfigResponse) {}
+
+    rpc PutClusterConfig(PutClusterConfigRequest) returns (PutClusterConfigResponse) {}
+}
+
+message RequestHeader {
```

Comment:
Can we remove it and use grpc's Header instead? https://github.com/grpc/grpc-go/blob/master/rpc_util.go#L172